src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/NcksToCsv.java

```diff
@@ -0,0 +1,176 @@
+package au.org.emii.gogoduck.worker;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class NcksToCsv {
+
+    public static Integer lineNumber = 0;
+    public static Integer attributesEnd = 0;
+    public static List<String> lines = new ArrayList<String>();
+    public static List<String> metadata = new ArrayList<String>();
+    public static List<String> attributeMetadata = new ArrayList<String>();
+    public static List<Map> attributes = new ArrayList<Map>();
+    public static StringBuilder csvString = new StringBuilder();
+
+    private static final String fNEW_LINE = System.getProperty("line.separator");
+    private static final String fCOMMENT = "# ";
+
+    public static void main (String... aArguments) {
```

Comment:
Tidy up this file. No new lines after function declarations, and also no space after function name before brackets.